src/services/contracts/codegen/createWatchContractEvent.ts

```diff
@@ -61,6 +61,6 @@ export function createWatchContractEvent<
       ...(addressOverride ? { address: addressOverride } : {}),
       ...(c.eventName ? { eventName: c.eventName } : {}),
       abi: c.abi,
-    }) as any; // TODO
+    }) as any;
```

Comment:
Should we use `any` here?

Very good job, tokens are fetches as well after adding a new one like 1,2,3 :)

Reply:
I think `any` was needed for the code generations we've used. Let's consider it a bit of the legacy already and see if we can come up with better and less custom combination of viem and walletconnect combo in the new app.